src/hb-ot-layout-common.hh

```diff
@@ -1574,15 +1574,25 @@ static void ClassDef_remap_and_serialize (hb_serialize_context_t *c,
                                           const hb_set_t &glyphset,
                                           const hb_map_t &gid_klass_map,
                                           hb_sorted_vector_t<HBGlyphID> glyphs,
```

Comment:
This one can be changed to reference to so the glyphs vector doesn't need to be copied.